build/build-utils.js

```diff
@@ -0,0 +1,69 @@
+import {
+    getIncludedExchangeIds,
+    exportExchanges,
+    createExchanges,
+    exportSupportedAndCertifiedExchanges,
+    exportExchangeIdsToExchangesJson,
+    exportKeywordsToPackageJson,
+    createMarkdownTable,
+    createMarkdownExchange,
+    createMarkdownListOfExchanges,
+    getReferralUrlOrWebsiteUrl,
+    getFirstWebsiteUrl,
+    getVersionBadge,
+    createMarkdownListOfCertifiedExchanges,
+    getFirstDocUrl,
+    getVersion,
+    getVersionLink,
+    createMarkdownListOfExchangesByCountries,
+    cloneGitHubWiki,
+    exportWikiToGitHub
+} from './export-helper.js'
+
+import {
+    vss,
+    vssEverything,
+} from './vss.js'
+
+import {
+    replaceInFile,
+    copyFile,
+    overwriteFile,
+    createFolder,
+    createFolderRecursively,
+} from './fsLocal.js'
+
+import {
+    Transpiler
+} from './transpile.js'
+
+// Create subpackage to be used on Pro side
```

Comment:
can you explain a bit more how this works and what are the advantages over just putting this in the top of a file

Reply:
@frosty00 
Ok not totally sure I understood the question but I will try to answer it.

Previously, if we needed to access a script from ccxt we could just specify the path in the require.

Example: 
```Javascript
const Transpiler = require ('ccxt/build/transpile.js')
const fs = require ('ccxt/build/fs.js')
```
Now we can't do this, we must explicitly define in `package.json` which "subPackages" the ccxt package contains and are accessible. To avoid having to specify a lot of sub-packages and most of them with a single file only (vss, transpile, etc) I decided to group all of these "utilities" in this file and expose only the "buildUtils" subpackage.

Reply:
ok, but why not just import it from ccxt, i'd rather avoid making a submodule if possible since it will pollute the ccxt namespace unnecessarily. I'd rather do

```
import {
       vss,
       Transpiler,
        getIncludedExchangeIds,
        exportExchanges,
        createExchanges,
        exportSupportedAndCertifiedExchanges,
        exportExchangeIdsToExchangesJson,
        exportKeywordsToPackageJson,
    } from  'ccxt';
 ```
 
 at the top of the ccxtpro file and delete the buildUtils file

Reply:
@frosty00 Because doing that will add those functions to the cjs and browser bundle and since the users will never need them I thought that would be better to keep them separated